docs/guides/creating-abilities.md

```diff
@@ -58,32 +58,116 @@ wp_register_ability('my-plugin/my-ability', [
 
 ## MCP Annotations
 
-Annotations provide behavior hints to MCP clients about how to handle your abilities. **All component types** (Tools, Resources, and Prompts) support annotations through the `meta.annotations` field:
+Annotations provide behavior hints to MCP clients about how to handle your abilities. **Annotations are type-specific** - Tools use different annotations than Resources and Prompts.
+
+### Annotation Format: WordPress Abilities API vs MCP
+
+**Best Practice: Use WordPress Abilities API Format**
+
+The MCP Adapter automatically converts WordPress Abilities API annotation names to MCP format. **It's recommended to use the WordPress Abilities API format** when available for consistency across the WordPress ecosystem.
+
+#### For Tools: WordPress Format Preferred
+
+```php
+// ✅ RECOMMENDED: WordPress Abilities API format
+'meta' => [
+    'annotations' => [
+        'readonly' => true,        // Auto-converted to readOnlyHint
+        'destructive' => false,    // Auto-converted to destructiveHint
+        'idempotent' => true,      // Auto-converted to idempotentHint
+        'openWorldHint' => false,  // No WordPress equivalent, use MCP format
+        'title' => 'My Tool'       // No WordPress equivalent, use MCP format
+    ]
+]
+
+// ✅ ALSO VALID: Direct MCP format
+'meta' => [
+    'annotations' => [
+        'readOnlyHint' => true,
+        'destructiveHint' => false,
+        'idempotentHint' => true,
+        'openWorldHint' => false,
+        'title' => 'My Tool'
+    ]
+]
+```
+
+**Tool Annotation Mapping Table:**
+
+| WordPress Format | MCP Format | Description |
+|-----------------|------------|-------------|
+| `readonly` | `readOnlyHint` | Tool doesn't modify data |
+| `destructive` | `destructiveHint` | Tool may delete/destroy data |
+| `idempotent` | `idempotentHint` | Same input → same output |
+| *(no equivalent)* | `openWorldHint` | Can work with arbitrary data |
```

Comment:
I'm inclined to make this `open-world` as I could see it being a more broadly supported Ability annotation in the future. Providing the indication that an Ability make be accessing things beyond the website itself is useful.